packages/workspace/src/generators/library/library.ts

```diff
@@ -82,8 +82,8 @@ export function addLint(
   });
 }
 
-function updateLibTsConfig(tree: Tree, options: NormalizedSchema) {
-  updateJson(tree, join(options.projectRoot, 'tsconfig.lib.json'), (json) => {
+function updateTsConfig(tree: Tree, options: NormalizedSchema) {
+  updateJson(tree, join(options.projectRoot, 'tsconfig.json'), (json) => {
```

Comment:
Strict options are moved into `tsconfig.json` to apply them to both `*.ts` and `.spec.ts`.